airflow/providers/google/cloud/example_dags/example_bigquery_queries.py

```diff
@@ -199,3 +233,4 @@
         execute_insert_query >> get_data >> get_data_result >> delete_dataset
         execute_insert_query >> execute_query_save >> bigquery_execute_multi_query >> delete_dataset
         execute_insert_query >> [check_count, check_value, check_interval] >> delete_dataset
+        insert_query_job >> extract_job >> load_job
```

Comment:
```suggestion
        insert_query_job >> extract_job >> load_job >> delete_dataset
```
So we avoid deleting dataset before